diff -uprNw massivethreads-0.99-adws-alpha/src/myth_io.h massivethreads-0.99-adws-alpha-patched/src/myth_io.h
--- massivethreads-0.99-adws-alpha/src/myth_io.h	2021-01-31 21:26:08.270870000 +0900
+++ massivethreads-0.99-adws-alpha-patched/src/myth_io.h	2021-01-31 17:56:15.160000000 +0900
@@ -8,6 +8,7 @@
 #include <sys/socket.h>
 #include <stdint.h>
 #include <stdarg.h>
+#include <sys/select.h>
 
 #include "myth/myth.h"
 #include "myth_config.h"
diff -uprNw massivethreads-0.99-adws-alpha/src/myth_real.c massivethreads-0.99-adws-alpha-patched/src/myth_real.c
--- massivethreads-0.99-adws-alpha/src/myth_real.c	2021-01-31 21:26:08.350870000 +0900
+++ massivethreads-0.99-adws-alpha-patched/src/myth_real.c	2021-01-31 21:28:38.760870000 +0900
@@ -53,6 +53,10 @@
 #include <link.h>
 #endif
 
+#ifdef pthread_equal
+#undef pthread_equal
+#endif
+
 
 typedef struct {
   int (*pthread_create)(pthread_t *thread, const pthread_attr_t *attr,
diff -uprNw massivethreads-0.99-adws-alpha/src/myth_wrap_pthread.c massivethreads-0.99-adws-alpha-patched/src/myth_wrap_pthread.c
--- massivethreads-0.99-adws-alpha/src/myth_wrap_pthread.c	2021-01-31 21:26:08.350870000 +0900
+++ massivethreads-0.99-adws-alpha-patched/src/myth_wrap_pthread.c	2021-01-31 21:29:20.240870000 +0900
@@ -13,6 +13,10 @@
 #include "myth_sched_func.h"
 #include "myth_sync_func.h"
 
+#ifdef pthread_equal
+#undef pthread_equal
+#endif
+
 static int myth_should_wrap_pthread(void) {
   static int s_myth_should_wrap_pthread = -1;
   if (s_myth_should_wrap_pthread == -1) {
